pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -2,238 +2,222 @@
 
 ```sh
-composer require magento/module-tax-sample-data-venia:dev-master
+magento-cloud checkout staging
 ```
 
-```sh
-composer require magento/module-sales-sample-data-venia:dev-master
-```
+## Add required Magento extensions
 
+Magento Cloud does not support node processes, so you cannot use UPWARD-JS to serve your storefront project.
+You must use the [magento2-upward-connector][] module with [UPWARD-PHP][] to deploy your storefront in Magento Cloud.
 
-These commands modify the `composer.json` file and adds the sample data modules to the `require` section:
+Use the `composer` CLI command to add the magento2-upward-connector module to the Magento installation:
 
-```json
-"magento/module-catalog-sample-data-venia": "dev-master",
-"magento/module-configurable-sample-data-venia": "dev-master",
-"magento/module-customer-sample-data-venia": "dev-master",
-"magento/module-tax-sample-data-venia": "dev-master",
-"magento/module-sales-sample-data-venia": "dev-master",
-"magento/sample-data-media-venia": "dev-master",
+```sh
+composer require magento/module-upward-connector
 ```
 
-## Install dependencies
+This command modifies the `composer.json` file and adds the package entry to the `require` section of the `composer.json` file.
 
-### Composer dependencies
+```text
+"magento/module-upward-connector": "^1.1.2"
+```
 
-_Optional_: If you manually modified the `composer.json` file to make the changes described in the previous steps, update your `composer.lock` with these new dependencies:
+{: .bs-callout .bs-callout-info}
+UPWARD-PHP is a dependency of the magento2-upward-connector, so
+you do not need to add it manually to your project.
 
-```sh
-composer update
-```
+## Set environment variables
 
-## Specify Cloud server environment
+PWA Studio storefronts require you to set the following [environment variables][] in your project for the build and runtime processes:
 
-{: .bs-callout .bs-callout-info}
-If you build your project locally and check the build artifacts into your Magento Cloud project, skip the **Install NPX and Yarn** and **Update build hooks** sections.
+| Name                                 | Buildtime                                     | Runtime                                       | Description                                                       |
+| ------------------------------------ | --------------------------------------------- | --------------------------------------------- | ----------------------------------------------------------------- |
+| `CONFIG__DEFAULT__WEB__UPWARD__PATH` |                                               | <i class="material-icons green">check_box</i> | Absolute path to UPWARD YAML configuration                        |
+| `NODE_ENV`                           | <i class="material-icons green">check_box</i> | <i class="material-icons green">check_box</i> | Specifies the node environment type                               |
```

Comment:
@tjwiebell does `NODE_ENV=production` really needed, given its part of script as well?

```
...... $ yarn build
yarn run v1.22.4
$ yarn venia run build
$ yarn workspace @magento/venia-concept run build
$ yarn run build:prod
$ yarn run clean && webpack --no-progress --env.mode production
```

Reply:
@jcalcaben You can remove remove this `NODE_ENV` variable, tested without it and it works.

Reply:
let's keep this line in, so people are aware it's a variable they can set. for example, they may want to specify 'staging' as the environment